partner_industry_secondary/views/res_partner_view.xml

```diff
@@ -75,16 +75,15 @@
         <field name="name">Partner form with industry</field>
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="base.view_partner_form" />
-        <field
-            name="groups_id"
-            eval="[(4, ref('partner_industry_secondary.group_use_partner_industry_for_person'))]"
-        />
         <field name="arch" type="xml">
             <xpath
                 expr="//page/group/group/field[@name='industry_id']"
                 position="attributes"
             >
                 <attribute name="attrs">{'invisible': 0}</attribute>
+                <attribute
+                    name="groups"
+                >!partner_industry_secondary.group_use_partner_industry_for_person</attribute>
```

Comment:
Not sure to fully understand. why `<attribute name="attrs">{'invisible': 0}` and just after `<attribute                     name="groups" >!partner_industry_secondary.group_use_partner_industry_for_person</a...`

Each node deal with visibility !? could it expressed with only one node ?

Reply:
I realize my solution does not work actually. I really struggle to understand this point of the migration documentation :

>Any view with groups_id on it now has to move such groups to the elements of the view. But now you can put groups attribute in a view field instead of isolating it on a view without fear of an access error. Check both things in https://github.com/odoo/odoo/pull/98551 and https://github.com/odoo/odoo/pull/95729.

Reply:
The issue is clear for me now : the attribute '{'invisible': [('is_company', '=', False)]}' must be removed when the user has the group `partner_industry_secondary.group_use_partner_industry_for_person`. I tried to do the following `<attribute name="attrs" groups="partner_industry_secondary.group_use_partner_industry_for_person">{'invisible': 0}</attribute>`, but it doesn't work (have no effect on attribute element ?). A solution would be to add as second OR condition in '{'invisible': [('is_company', '=', False)]}' `ref(partner_industry_secondary.group_use_partner_industry_for_person) =True`. But I don't know how to call a ref() in a domain in a view.

Reply:
> But I don't know how to call a ref() in a domain in a view.

You can't

Reply:
`%(my_module.myidstring)d` is evaluated in view, I thought before insertion in database !?

Reply:
yes but a domain ["(%(my_module.myidstring)d", "=", True)] doesn't seem to work